wip-0009.md

```diff
@@ -61,14 +61,38 @@ Upon reception of multiple candidates:
 2. Block candidates from active identities are prioritized over those from non-active identities.
 3. If there are still multiple candidates, the one with the lowest VRF value is chosen.
 
-With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies.
+With the previous changes, the newcomer mining probability is bounded as initially designed in [RandPoe] while the fair distribution among those identities that have proven some contribution to the network's service and security still applies. Just as an illustrative example with the aforementioned network conditions, the newcomer mining probability would be almost 5%.
+
+
+## Backwards compatibility
+
+### Consensus
+
+This proposal introduces a significant change in the block candidate selection algorithm. Therefore there is enough evidence to assert that it is not backward compatible from the consensus perspective and that no cooperative chain construction would be possible among two nodes from which one adopted these changes while the other did not.
+
+As it is considered best practice in such situations, an *Adoption Plan* is hereby proposed.
+
+
+### Examples and Documentation
+
+The algorithm affected by this proposal is internal to the block chain protocol, and it does not trigger any update on existing examples or documentation.
+
+
+### Libraries and Clients
+
+The algorithm affected by this proposal is internal to the block chain protocol, and it does not require any update on existing libraries or clients.
 
 
 ## Reference Implementation
 
 The business logic described in the specification has been implemented in [witnet-rust] by [pull request #1829][witnet/witnet-rust#1829].
 
 
+## Adoption Plan
+
+The changes proposed herein have been already implemented in [witnet-rust] and the activation of the new consensus rule is scheduled as a community-coordinated hard fork to take place on March 16 2021 at 9 am UTC, as proposed by [pull request #1829][witnet/witnet-rust#1829] and [widely announced to the community][announcement] in a timely manner.
```

Comment:
`announcement` link does not exist.

Reply:
Ups sorry, I forgot to include the medium post link